src/utils/errorHandlers.js

```diff
@@ -0,0 +1,23 @@
+const errorHandler = (req, res, err) => {
+    if (err.statusCode) {
+        res.status(statusCode).json({
+            status: 'error',
```

Comment:
status should bear the status code

Reply:
Yes the statusCode is a user given error code, that is if I inserted the error code myself, and other follows like that and the last one is generated by the server